src/app/icons/icons-sets/icons-sets.component.ts

```diff
@@ -148,9 +148,11 @@ export class IconsSetsComponent implements AfterViewInit, OnDestroy {
     filterIconsOnSearch() {
 
         this.subscriptions.push(this._iconsViewService.searchValue
-            .debounceTime(150)
+            .debounceTime(200)
```

Comment:
👍

Reply:
My understanding is that we wanted to ping Google Analytics everytime someone searched for an icon. Is it possible for us to add that functionality?